R/utils.R

```diff
@@ -422,14 +426,49 @@ check_results_names <- function(names, method_name) {
     dup_names <- unique(names[duplicated(names)])
     abort(
       paste0(
-        "Cannot create results tibble with duplicate column names: `",
+        "Cannot create `fit_results` tibble with duplicate column names: `",
```

Comment:
- [ ] Check out [new test failure](https://github.com/Yu-Group/simChef/actions/runs/3767850990/jobs/6405777892) at this line: https://github.com/Yu-Group/simChef/blob/b3d71a2d5594eab4f8f8ec1006aba8e723508b68/tests/testthat/test-parallel-strategies.R#L193